package.json

```diff
@@ -60,6 +60,7 @@
     "deep-freeze": "^0.0.1",
     "eventemitter3": "^5.0.1",
     "express-session": "^1.18.1",
+    "json-schema-to-ts": "3.1.1",
```

Comment:
This needs to become an actual dependency (rather than dev dependency) as `addTaskHandler` is exposed externally and depends on it